patches/components-omnibox-browser-omnibox_controller.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/omnibox/browser/omnibox_controller.h b/components/omnibox/browser/omnibox_controller.h
+index 1e1d605e64a010f382d62ea88c51f82122a3b872..4c9284f31e2545797af039e22111004f6377de15 100644
+--- a/components/omnibox/browser/omnibox_controller.h
++++ b/components/omnibox/browser/omnibox_controller.h
+@@ -36,6 +36,7 @@ class OmniboxController : public AutocompleteControllerDelegate {
+   ~OmniboxController() override;
+ 
+   // The |current_url| field of input is only set for mobile ports.
++  virtual
```

Comment:
you can add `virtual` using a chromium_src override

Reply:
Oh! I didn't know c++ allows`virtual` keyword to appear both before and after the return type. :+1: 
Done. Thanks.